.storybook/main.js

```diff
@@ -55,7 +55,7 @@ module.exports = {
       }),
       new webpack.EnvironmentPlugin({
         EMBEDDING_SDK_VERSION,
-        IS_EMBEDDING_SDK_BUILD: isEmbeddingSDK,
+        IS_EMBEDDING_SDK: isEmbeddingSDK,
```

Comment:
we had both "IS_EMBEDDING_SDK" and "IS_EMBEDDING_SDK_BUILD" which was confusing, I changed so we use only one